src/malos.cpp

```diff
@@ -88,12 +87,6 @@ int RunServer() {
   }
   driver_manager.RegisterDriver(&driver_uv);
 
-  ZigbeeDriver driver_zigbee;
-  if (!driver_zigbee.Init(kBasePort + 4 * 5 + 1, kUnsecureBindScope)) {
```

Comment:
A comment is good here:

// kBasePort + 4 * 5 + 1 is reserved to the ZigBee driver, and it graduate to its own repo.